roles/common/tasks/main.yml

```diff
@@ -20,6 +20,37 @@
     retries: 5
   when: ansible_distribution == "Ubuntu"
 
+- name: Register with RHN
```

Comment:
Need to create a separate role for rhn-subscription and move logic of subscription, attaching to pool, disbale all repos )only flotsam) and enabled desire repos.